exchange.idl

```diff
@@ -2172,12 +2177,16 @@ System Attendant Private Interface
 		[switch_is(1)] GLOBCNT	GLOBCNT;
 	} ShortTermID; /* MID, FID, CN */
 
-	typedef [public,flag(NDR_NOALIGN)] struct {
-		GUID			GUID;
-		uint8			Size;
-		uint8			Data[Size];
+	typedef [public,noprint,flag(NDR_NOALIGN)] struct {
+		GUID			NameSpaceGuid;
+		[flag(NDR_REMAINING)] DATA_BLOB LocalId;
```

Comment:
This change makes SOGo-OpenChange library not compile.
```
MAPIStoreMailFolder.m:559:40: error: ‘struct XID’ has no member named ‘GUID’
   guid = [NSString stringWithGUID: &xid->GUID];
 ```
An update in SOGo library is required.